collectors/python.d.plugin/alarms/README.md

```diff
@@ -19,7 +19,7 @@ This collector creates an 'Alarms' menu with one line plot showing alarm states
 
 Below is an example of the chart produced when running `stress-ng --all 2` for a few minutes. You can see the various warning and critical alarms raised. 
 
-![alt text](https://github.com/andrewm4894/random/blob/master/images/netdata/netdata-alarms-collector.jpg)
+![alarms collector](https://user-images.githubusercontent.com/2178292/101505477-4f9ef200-396c-11eb-864c-a44d9eaf2ee5.png)
```

Comment:
as i mentioned https://github.com/netdata/netdata/pull/10042#issuecomment-740702959

this image is not up to date:
 - the title has not status mapping
 - wrong icon (❗ => 🔔 )
 - wrong priority - the chart below `Netdata Monitoring`

Reply:
This looks to be fixed following the suggested update from @joelhans